packages/layout/src/components/beta/beta.tsx

```diff
@@ -1,15 +1,12 @@
-import React, { useMemo } from 'react';
-import { AppWidth, DocWidth, Flex, P } from '@tpr/core';
+import React from 'react';
+import { AppWidth, DocWidth, Flex, Link, P } from '@tpr/core';
 import style from './beta.module.scss';
 
 type BetaHeaderProps = {
-	text: any;
+	text?: string;
 };
-export const BetaHeader: React.FC<BetaHeaderProps> = ({ text }) => {
-	const TextComponent = useMemo(() => {
-		return typeof text === 'function' ? text : () => text;
-	}, [text]);
 
+export const BetaHeader: React.FC<BetaHeaderProps> = ({ text }) => {
```

Comment:
Definitely want to keep using either useMemo or React.Memo higher order component here for performance.

Reply:
Reverted to using useMemo with HOC